jail-ftp.yml

```diff
@@ -11,7 +11,6 @@
   - base
   - role: ftp
     ftp_root: "{{ ftp_root }}"
-    ftp_hostname: "{{ ftp_hostname }}"
```

Comment:
I wonder why it's not complaining about `ftp_root`. Could be because it's checking alphabetically so `ftp_hostname` errors first. Seems likely that both will have issues.